src/components/RecoveryWords.css

```diff
@@ -28,23 +28,20 @@
     line-height: 4rem;
     will-change: transform;
```

Comment:
Yep, from that page, will-change can use more resources and make the page slower if used often
> Don't apply will-change to elements to perform premature optimization. If your page is performing well, don't add the will-change property to elements just to wring out a little more speed. will-change is intended to be used as something of a last resort, in order to try to deal with existing performance problems.

Reply:
Removed here: https://github.com/nimiq/keyguard-next/pull/119/commits/1efa07cd7e3eb91e5a3b1a27b7c5286052d17060#diff-965ca520b4b6a1ba6f3338f676f33d23

Reply:
there are some other occurrences in our code base and i think the case where it actually helps us is rare. Let's look into it tomorrow, but not as part of this pr.

Reply:
I think the only remaining are in the `IdenticonSelector`s for the transforming Identicons, address, and background-cover. It's kind-of ok there, because the click of the user selecting an identicon is not predictable (except for hover states). But maybe we don't even need it there either.

Reply:
In https://github.com/nimiq/vue-components a bunch of them is in the code.